src/main/java/org/betonquest/betonquest/modules/config/ConfigurationFileImpl.java

```diff
@@ -85,8 +88,28 @@ private static ConfigAccessor createPatchAccessor(final Plugin plugin, final Str
     }
 
     @SuppressWarnings("PMD.UnusedFormalParameter")
-    private boolean patchConfig(final ConfigurationSection patchAccessorConfig) {
-        return false;
+    private void patchConfig(final ConfigurationSection patchAccessorConfig) {
+        final Patcher patcher = new Patcher(accessor.getConfig(), patchAccessorConfig);
+        if (patcher.hasUpdate()) {
+
+            final String configName = accessor.getConfigurationFile().getName();
+            final String currentVersion = accessor.getConfig().getString("configVersion", "2.0.0-CONFIG-0");
+            LOG.info("Patch for configuration '%s' with current version '%s' found.".formatted(configName, currentVersion));
+            LOG.info("Backing up current config...");
+
+            final File dataFolder = BetonQuest.getInstance().getDataFolder();
+            Utils.backup(dataFolder, accessor.getConfig(), false);
```

Comment:
Only one global backup? or a backup for every patch?
also is the path here correct?

Reply:
Backups are not part of this PR anymore